crates/astria-composer/src/searcher.rs

```diff
@@ -1,7 +1,13 @@
-use std::time::Duration;
+use std::{
+    sync::Arc,
+    time::Duration,
+};
 
 use astria_sequencer::{
-    accounts::types::Nonce,
+    accounts::types::{
```

Comment:
Nit: not a fan of the increasing coupling between sequencer and composer here. I am preparing a PR to reduce this.

Reply:
We should think about what types in sequencer are things that we and others might want to use externally and create a crate for these. Things like nonce, transactions and signing them are types of things external tools might build around.